test/format/format_inline.h

```diff
@@ -385,8 +388,8 @@ lock_readlock(WT_SESSION *session, RWLOCK *lock)
 }
 
 /*
- * lock_writeunlock --
- *     Release an exclusive lock.
+ * lock_readunlock --
```

Comment:
Updated function description should be (see `__wt_readunlock`):
```
/*
 * lock_readunlock --
 *     Release a shared lock.
 */
```

Reply:
addressed here: [898d602](https://github.com/wiredtiger/wiredtiger/pull/8616/commits/898d602a18fbcf806e8a8f743874def814dcb160)